backend/dataall/modules/dataset_sharing/services/share_managers/s3_bucket_share_manager.py

```diff
@@ -72,95 +73,71 @@ def grant_s3_iam_access(self):
         logger.info(
             f'Grant target role {self.target_requester_IAMRoleName} access policy'
         )
-        existing_policy = IAM.get_role_policy(
+        bucket_policy_name = ConsumptionRole.generate_policy_name(self.target_environment.environmentUri,
```

Comment:
can we name `bucket_iam_policy_name` or something that indicates this is the iam policy not the s3 bucket policy

Reply:
changed